docs/model_zoo.md

```diff
@@ -13,6 +13,30 @@ You can replace `https://s3.ap-northeast-2.amazonaws.com/open-mmlab` with `https
 - For fair comparison with other codebases, we report the GPU memory as the maximum value of `torch.cuda.max_memory_allocated()` for all 8 GPUs. Note that this value is usually less than what `nvidia-smi` shows.
 - We report the inference time as the total time of network forwarding and post-processing, excluding the data loading time. Results are obtained with the script [benchmark.py](https://github.com/open-mmlab/mmdetection/blob/master/tools/analysis_tools/benchmark.py) which computes the average time on 2000 images.
 
+## ImageNet Pretrained Models
+
+It is common to initialize from backbone models pre-trained on ImageNet classification task. All pre-trained  model links can be found at [open_mmlab](https://github.com/open-mmlab/mmcv/blob/master/mmcv/model_zoo/open_mmlab.json).  According to `img_norm_cfg`, we can divide all the ImageNet  pre-trained  model weights into four cases:
+
+- TorchVision styles:  Corresponding to torchvision weight, including ResNet50, ResNet101, RegNetX_3.2gf、ResNext101_32x4d, and ResNext101_64x4d. The `img_norm_cfg` is `dict(
```

Comment:
RegNetX_3.2gf should be all the regnetx models? And we may also add them in the table below.